docs/get-started/scaffolding.md

```diff
@@ -0,0 +1,34 @@
+---
+id: scaffolding
+title: Scaffolding
+sidebar_label: Scaffolding
+---
+
+# Webiny Scaffold CLI
+
+### Introduction
+Sometimes when you develop applications with Webiny you will end up spending time doing common tasks that can easily become repetitive and tedious over time. This is where `webiny scaffold` comes into play.
+
+### Usage
+You will of course need to install Webiny's CLI if you haven't done so already: `npm -g install @webiny/cli`.
```

Comment:
Not only that, you need to create a project. So I would rephrase this sentence into something like `This assumes you already have a Webiny project to work on. If not, see the getting started guide (link to guide)`.

Reply:
(done a while ago; checking over again right now)